modules/cert-manager-override-scheduling.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-override-scheduling_{context}"]
+= Configuring scheduling overrides for cert-manager components
+
+After installing the {cert-manager-operator}, you can configure the pod scheduling from the {cert-manager-operator} API for the {cert-manager-operator} components such as cert-manager controller, CA injector, and Webhook. 
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have installed version 1.15.0 or later of the {cert-manager-operator}.
+
+.Procedure
+
+* Update the `certmanager.operator` custom resource to configure pod scheduling overrides for the desired components. Use the `overrideScheduling` field under `controllerConfig`, `webhookConfig`, or `cainjectorConfig` to define `nodeSelector` and `tolerations` settings.
++
+.Example output
+[source,yaml]
+----
+$ oc patch certmanager.operator cluster --type=merge -p="
+spec:
+  controllerConfig:
+    overrideScheduling:
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: '' <1>
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule <2>
+  webhookConfig:
+    overrideScheduling:
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: '' <3>
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule <4>
+  cainjectorConfig:
+    overrideScheduling:
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: '' <5>
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule <6>
+----
+<1> Defines the `nodeSelector` for the cert-manager controller deployment.
+<2> Defines the `toleration` for the cert-manager controller deployment.
+<3> Defines the `nodeSelector` for the cert-manager webhook deployment.
+<4> Defines the `toleration` for the cert-manager webhook deployment.
+<5> Defines the `nodeSelector` for the cert-manager cainjector deployment.
+<6> Defines the `toleration` for the cert-manager cainjector deployment.
+
+
+.Verification
+
+. Verify pod scheduling settings for `cert-manager` pods:
+
+.. Check the deployments in the `cert-manager` namespace to confirm they have the correct `nodeSelector` and `tolerations`:
```

Comment:
As mentioned before, steps that introduce commands need to specifically mention that:
```suggestion
.. Check the deployments in the `cert-manager` namespace to confirm they have the correct `nodeSelector` and `tolerations` by running the following command:
```